### PR TITLE
[DOC]Add saslJaasServerRoleTokenSignerSecret parameter in security-kerberos.md

### PR DESCRIPTION
**Pull request by liudezhi2098**
Master Issue: #15121

Preview looks good:
image

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 
  
- [x]  `doc`

### CODE REVIEW

**Comment by liudezhi2098** [comment]
@momo-jun PTAL

File: site2/docs/reference-configuration.md [hunk 1]
```diff
@@ -598,6 +598,7 @@ You can set the log level and configuration in the  [log4j2.yaml](https://github
 | tokenAudience | The token audience stands for this broker. The field `tokenAudienceClaim` of a valid token need contains this parameter.| |
 |saslJaasClientAllowedIds|This is a regexp, which limits the range of possible ids which can connect to the Broker using SASL. By default, it is set to `SaslConstants.JAAS_CLIENT_ALLOWED_IDS_DEFAULT`, which is ".*pulsar.*", so only clients whose id contains 'pulsar' are allowed to connect.|N/A|
 |saslJaasServerSectionName|Service Principal, for login context name. By default, it is set to `SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME`, which is "Broker".|N/A|
+|saslJaasServerRoleTokenSignerSecret|Secret for SaslRoleTokenSigner. By default, it is set to `SaslConstants.JAAS_DEFAULT_ROLE_TOKEN_SIGNER_SECRET`, which is "PulsarSecret".|N/A|
```
Comment (momo-jun) [inline site2/docs/reference-configuration.md:601]: Should the default value be put in the last column?

File: site2/docs/reference-configuration.md [hunk 2]
```diff
@@ -598,6 +598,7 @@ You can set the log level and configuration in the  [log4j2.yaml](https://github
 | tokenAudience | The token audience stands for this broker. The field `tokenAudienceClaim` of a valid token need contains this parameter.| |
 |saslJaasClientAllowedIds|This is a regexp, which limits the range of possible ids which can connect to the Broker using SASL. By default, it is set to `SaslConstants.JAAS_CLIENT_ALLOWED_IDS_DEFAULT`, which is ".*pulsar.*", so only clients whose id contains 'pulsar' are allowed to connect.|N/A|
 |saslJaasServerSectionName|Service Principal, for login context name. By default, it is set to `SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME`, which is "Broker".|N/A|
```
Comment (momo-jun) [inline site2/docs/reference-configuration.md:600]: This default value is different from the one shown in the `security-kerberos` file, which is "PulsarBroker". Which one is correct?

File: site2/docs/security-kerberos.md [hunk 3]
```diff
@@ -114,7 +114,8 @@ You can have 2 separate JAAS configuration files:
  - Set `authenticationProviders` to choose `AuthenticationProviderSasl`;
  - Set `saslJaasClientAllowedIds` regex for principal that is allowed to connect to broker;
  - Set `saslJaasServerSectionName` that corresponds to the section in JAAS configuration file for broker;
- 
+ - Set `saslJaasServerRoleTokenSignerSecret` that corresponds to role token signer secret in JAAS configuration file for broker;
```
Comment (momo-jun) [inline site2/docs/security-kerberos.md:117]:
```suggestion
 - Set `saslJaasServerRoleTokenSignerSecret` that corresponds to the secret of role token signer in JAAS configuration file for broker;
```

File: site2/docs/reference-configuration.md (hunk 2 above)
Comment (liudezhi2098) [reply site2/docs/reference-configuration.md:600]: PulsarBroker is correct

File: site2/docs/reference-configuration.md [hunk 4]
```diff
@@ -597,7 +600,8 @@ You can set the log level and configuration in the  [log4j2.yaml](https://github
 |tokenAudienceClaim| The token audience "claim" name, e.g. "aud". It is used to get the audience from token. If it is not set, the audience is not verified. ||
 | tokenAudience | The token audience stands for this broker. The field `tokenAudienceClaim` of a valid token need contains this parameter.| |
 |saslJaasClientAllowedIds|This is a regexp, which limits the range of possible ids which can connect to the Broker using SASL. By default, it is set to `SaslConstants.JAAS_CLIENT_ALLOWED_IDS_DEFAULT`, which is ".*pulsar.*", so only clients whose id contains 'pulsar' are allowed to connect.|N/A|
-|saslJaasServerSectionName|Service Principal, for login context name. By default, it is set to `SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME`, which is "Broker".|N/A|
+|saslJaasServerSectionName|Service Principal, for login context name. By default, it is set to `SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME`, which is "PulsarBroker".||
```
Comment (momo-jun) [inline site2/docs/reference-configuration.md:603]: Need to add "N/A" in the last column.

**Review by momo-jun — APPROVED** [review]

File: site2/docs/reference-configuration.md [hunk 5]
```diff
@@ -243,6 +243,9 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
 |tokenAuthClaim| Specify which of the token's claims will be used as the authentication "principal" or "role". The default "sub" claim will be used if this is left blank ||
 |tokenAudienceClaim| The token audience "claim" name, e.g. "aud", that will be used to get the audience from token. If not set, audience will not be verified. ||
 |tokenAudience| The token audience stands for this broker. The field `tokenAudienceClaim` of a valid token, need contains this. ||
+|saslJaasClientAllowedIds|This is a regexp, which limits the range of possible ids which can connect to the Broker using SASL. By default, it is set to `SaslConstants.JAAS_CLIENT_ALLOWED_IDS_DEFAULT`, which is ".*pulsar.*", so only clients whose id contains 'pulsar' are allowed to connect.|N/A|
```
Comment (Anonymitaet) [inline site2/docs/reference-configuration.md:246]:
```suggestion
|saslJaasClientAllowedIds|This is a regexp, which limits the range of possible IDs. These IDs can connect to the broker using SASL. By default, it is set to `SaslConstants.JAAS_CLIENT_ALLOWED_IDS_DEFAULT`, which is ".*pulsar.*", so only clients whose ID contains 'pulsar' are allowed to connect.|N/A|
```

File: site2/docs/reference-configuration.md [hunk 6]
```diff
@@ -243,6 +243,9 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
...
+|saslJaasServerSectionName|Service Principal, for login context name. By default, it is set to `SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME`, which is "PulsarBroker".|N/A|
```
Comment (Anonymitaet) [inline site2/docs/reference-configuration.md:247]:
```suggestion
|saslJaasServerSectionName|Service Principal, for login context name. By default, it is set to `SaslConstants.JAAS_DEFAULT_BROKER_SECTION_NAME`, which is "PulsarBroker".|N/A|
```

what's the meaning of "Service Principal, for login context name. "?

Comment (liudezhi2098) [reply site2/docs/reference-configuration.md:247]: This is the concept of kerberos，principal can be understood as the user's identity, will bring this identity when you log in

**Review by Anonymitaet — APPROVED** [review]

**Comment by momo-jun** [comment]
Hi @liudezhi2098 , I noticed that the master code PR was merged two days ago. Does this doc PR need any updates? 
If it is good to go, ping @Anonymitaet for a check and merge.

**Comment by liudezhi2098** [comment]
> Hi @liudezhi2098 , I noticed that the master code PR was merged two days ago. Does this doc PR need any updates? If it is good to go, ping @Anonymitaet for a check and merge.

need review again, thx.

**Comment by Anonymitaet** [comment]
@liudezhi2098 I've renamed your PR based on the [Pulsar PR Naming Guideline](https://docs.google.com/document/d/1d8Pw6ZbWk-_pCKdOmdvx9rnhPiyuxwq60_TrD68d7BA/edit#heading=h.wu6ygjne8e35). Please follow it, thanks.

**Pull request merged**